src/app/content/highlights/components/ShowMyHighlightsStyles.tsx

```diff
@@ -3,13 +3,13 @@ import { AngleUp } from 'styled-icons/fa-solid/AngleUp';
 import { labelStyle } from '../../../components/Typography';
 import { h4Style } from '../../../components/Typography/headings';
 import theme from '../../../theme';
-import { PopupBody, popupBodyPadding, popupPadding } from './HighlightStyles';
 import {
   desktopHorizontalMargin,
   desktopVerticalMargin,
   mobileMarginSides,
   mobilePaddingSides
-} from './SummaryPopup/constants';
+} from '../../styles/constants';
```

Comment:
this constants file should probably be named `PopupConstants` or something less generic like that